scalanet/src/io/iohk/scalanet/peergroup/dynamictls/DynamicTLSPeerGroup.scala

```diff
@@ -113,6 +114,15 @@ class DynamicTLSPeerGroup[M](val config: Config)(
 }
 
 object DynamicTLSPeerGroup {
+  def getMessageQueue[M](typeOf: ChannelType = monix.execution.ChannelType.SPMC) = {
+    ConcurrentQueue.unsafe[Task, M](BufferCapacity.Unbounded(None), typeOf)
+  }
+
+  def pushEventOnNettyScheduler[M](queue: ConcurrentQueue[Task, M], ev: M)(implicit s: Scheduler): Unit = {
```

Comment:
I guess this was copy pasted, but obviously the naming here is misleading as the scheduler itself is implicit.

Reply:
yea, it should probably take event loop as param to make it clear